ecis

```diff
@@ -138,14 +160,22 @@ function catch_error {
 }
 
 case "$1" in
+
+    commit)
+        git_commit
+    ;;
+
+    git-all)
+        git add .
+        git_commit
+        current_branch=$(git branch | grep "*" | tr -d "()*" | cut -c2-)
```

Comment:
This command can be simplified using awk, for example, current_branch = $ (git branch | grep "*" | awk '{print $2}')